frontend/src/css/Account.styl

```diff
@@ -1,181 +1,44 @@
+@import '~tracim_frontend_lib/src/css/Variable.styl'
+
 .nomarginlabel
   margin-bottom 0
 
-settingText()
```

Comment:
Is this deadcode?

Reply:
Yes, I deleted everything that was not used.